app/crowd.yml

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
Need to clean up this file or create a separate tech debt ticket for this.
E.g. commented lines, correct crowd ports, default pass, etc.
Also, it makes sense to have another ticket for RPS approach?

Reply:
Already created appropriate task